cumulusci.yml

```
@@ -92,21 +91,22 @@ tasks:
             field: hed__Application_Type__c
             values: "Elementary School,Middle School,High School"
             existing_values: "Elementary School,Middle School,High School"
-            sorted: true
+            sorted: True
             namespaced: False
 
     add_behavior_response_type_values:
         description: Adds additional picklist values to the Type field.
-        class_path: tasks.add_picklist_values.AddPicklistValues
+        class_path: cumulusci.tasks.metadata_etl.picklists.AddPicklistEntries
```

```suggestion
        class_path: tasks.picklists.AddPicklistEntries
```
Please change the classpath on this for the moment.